.gitmodules

```diff
@@ -7,4 +7,4 @@
 [submodule "data"]
 	path = data
 	url = https://github.com/NBISweden/ejprd-data.git
-	branch = feat/beaconize_data
+	branch = main
```

Comment:
This is useful when you want to indicate where to pull in new changes from but it’s probably best to also update the reference to the commit that was used to make sure that everything will work the same.